atomic-write.cabal

```diff
@@ -96,11 +96,11 @@ test-suite atomic-write-test
 
   build-depends:       base >= 4.5 && < 5.0
                      , atomic-write
-                     , temporary
-                     , unix-compat
-                     , filepath
-                     , text
-                     , bytestring >= 0.10.4
+                     , temporary >= 1.3 && < 1.4
```

Comment:
@Alex0jk in general, adding boundaries to packages is a good practice. However, I'm not sure about duplicating the package boundaries here because the `test` block is dependent on `atomic-write`, so it would inherit all of the package boundaries from the `library` section. I’m just trying to make it easier to maintain these boundaries in the long run, but I think it would be good to validate this with other Haskellers. @CristhianMotoche, @jpvillaisaza any thoughts?

Reply:
That was something I was thinking about. I am good we removing them. It makes sense to not have them here. Let´s see what others say.

Reply:
Since `temporary` already has version constraints as a library dependency, there's no need to add it here. If you remove the constraints here and run `cabal check`, it shouldn't list `temporary` as missing bounds (if you remove them from the library component as well, it should).

I also wouldn't add constraints for boot libraries (`bytestring`, `directory`, `filepath`, `text`), but `cabal check` does complain about those.

Reply:
But consider using common stanzas as well if that's an option.